WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -432,12 +435,20 @@ class PostListMainViewModel @Inject constructor(
         )
     }
 
-    private fun showPrepublishingBottomSheet(post: PostModel) {
+    fun showPrepublishingBottomSheet(post: PostModel) {
```

Comment:
We might want to consider renaming this method to something else - it doesn't just "show bottom sheet" anymore. Wdyt?

AFAIK this method is used only when the user clicks on "Publish" button right? Shouldn't we always update the date to the current date?

Reply:
Good find. can this approach work? b51fd05

Reply:
Yeah, I think it should work. Thanks!

Reply:
Hmm so I was wrong. I think the `postStatus == DRAFT` condition needs to stay there. We show the publish button even on the published tab when the post has some local changes.

Btw I also noticed that when I change the post status in the post settings the change doesn't get stored. I think I already reported this, but mentioning it again just to be sure we didn't miss it.

![post-status-lost](https://user-images.githubusercontent.com/2261188/85681904-0c0d4000-b6cc-11ea-8a7d-76701e53c659.gif)

Reply:
Good find. I am wondering if this change is related to this project. I am going to investigate this because at first glance it seems unrelated.

Reply:
Definitely worth checking 👍.

Reply:
I can confirm it's not related to these changes and is actually expected behavior - https://github.com/wordpress-mobile/WordPress-Android/blob/978f3ab9d27f3dde01f1488fc7a8c7688e5f1416/WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadHandler.java#L630

Reply:
Awesome. Thanks for confirming 🙏